cutevariant/gui/vqleditor.py

```diff
@@ -112,20 +116,28 @@ def __init__(self):
         self.highlighter = VqlSyntaxHighlighter(self.text_edit.document())
         main_layout = QVBoxLayout()
         main_layout.addWidget(self.text_edit)
-        main_layout.setContentsMargins(0,0,0,0)
+        main_layout.setContentsMargins(0, 0, 0, 0)
         self.setLayout(main_layout)
         self.text_edit.textChanged.connect(self.changed)
         self._query = None
+        self.query_error = False
 
     @property
     def query(self):
         """Method override from AbstractQueryWidget"""
         try:
             self._query.from_vql(self.text_edit.toPlainText())
+            if self.query_error:
+                self.message.emit("")
             return self._query
```

Comment:
if the query is wrong, it doesn't have to return the query

Reply:
It is just a boolean that allows to reset the status bar only if there was a previous error on the query; in case of error, this function still returns None